packages/tocco-ui/src/Menu/ItemAccordion.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import Button from '../Button'
+import Item, {ItemStyles as ItemAccordionStyles} from './Item'
+
+class ItemAccordion extends Item {
+  render() {
+    return (
+      <ItemAccordionStyles
+        isOpen={this.state.isOpen}
+        isToggable={this.props.isToggable}
+      >
+        <Button
+          icon={this.state.isOpen ? 'fa-caret-up' : 'fa-caret-down'}
+          iconPosition="after"
+          label={this.props.label}
+          look={this.props.look}
+          onMouseDown={this.toogleOpenState}
+        />
+        {this.childs}
+      </ItemAccordionStyles>
+    )
+  }
+}
+
+Item.defaultProps = {
+  isOpen: true,
+  isToggable: true
```

Comment:
I'd use the term "toggleable" (183k google search results) instead of "toggable" (18k search results) ("togglable" = 98k search results)

Reply:
see 70afaf07d953a208b2f9843bda3dc76c17d5d5b2